src/backend/base/langflow/services/tracing/service.py

```diff
@@ -60,34 +60,23 @@ def __init__(self, settings_service: SettingsService):
         self.project_name: str | None = None
         self._tracers: dict[str, BaseTracer] = {}
         self._logs: dict[str, list[Log | dict[Any, Any]]] = defaultdict(list)
-        self.running = False
         self.end_trace_tasks: set[asyncio.Task] = set()
         self.deactivated = self.settings_service.settings.deactivate_tracing
         self.session_id: str | None = None
 
     async def start(self) -> None:
-        if self.running:
-            return
-        try:
-            self.running = True
-        except Exception:  # noqa: BLE001
-            logger.exception("Error starting tracing service")
+        pass
 
     async def flush(self) -> None:
         # This method is kept for API compatibility but now has minimal implementation
         try:
             # Wait for any pending trace tasks to complete
-            if self.end_trace_tasks:
-                await asyncio.gather(*self.end_trace_tasks, return_exceptions=True)
+            await asyncio.gather(*self.end_trace_tasks)
```

Comment:
Why not keep `return_exceptions`?

Reply:
If `return_exceptions` is True then any exception is catched and set in the result. So we don't get the `Error flushing logs` log.

Reply:
I see.